rngd.c

```diff
@@ -623,6 +623,8 @@ static error_t parse_opt (int key, char *arg, struct argp_state *state)
 	case 's':
 		if (sscanf(arg, "%i", &arguments->random_step) == 0)
 			argp_usage(state);
+		if (arguments->random_step > FIPS_RNG_BUFFER_SIZE || arguments->random_step < 0)
+			arguments->random_step = FIPS_RNG_BUFFER_SIZE
```

Comment:
There is a semicolon missing breaking the build.

Reply:
sure, a sec ...

Reply:
ok